### DISCUSSION TITLE
Docker won´t start

### DISCUSSION THREAD

**Discussion by matzeeg3** [isaacrlevin/presencelight#830] [Q&A]
Hi everyone,
i try to get the system running in an docker container on arm64.
the container starts up but it did not trust the certificate so it would not start.
and additional i don´t find informations where the app stores the settings after the container will run....
someone here can help me?
thanks in advanced
kind regards
matthias

**Comment by isaacrlevin** [comment]
I assume you are using the web version of the app. If you shell into the container, is there an appsettings.json file?

**Comment by matzeeg3** [comment]
hi isac, i don´t now why but now it is starting, but i have to do a ssh tunnel to authentikate with the app. i tryed it with the dns https://presencelight.local on my local dns but then i get an redirecte loop

**Reply by isaacrlevin** [reply]
So it is working?

**Reply by matzeeg3** [reply]
No I just get the redirect loop without any error in the browser. I have to to an ssh tunnel with port 5000 and 5001 and use localhost than it is working but that is not a good way for the application. Is there a way to set the answer url on the azure app you deploy? So I can use a let's encrypt cert? Should it work with the local DNS I set? If I go to https:: precentslight.local:5001 I directly get an login to aad and the if I log on the aad login directly comes back all the time.

**Reply by matzeeg3** [reply]
Did You know why this happened? What else url I can use? I can set any in my local DNS Server

**Reply by isaacrlevin** [reply]
I assume the error you are getting is related to the redirect Uri. I cannot add unlimited redirect uris. Here are the supported Uris. 
<img width="272" alt="image" src="https://github.com/isaacrlevin/presencelight/assets/8878502/37931bfa-2f0a-4f96-8c49-17e32a6da60f">

**Comment by isaacrlevin** [comment]
Closing as the Web/Container version does not currently work
